Sofa/framework/Type/src/sofa/type/RGBAColor.cpp

```diff
@@ -176,6 +177,30 @@ static std::istream& trimInitialSpaces(std::istream& in)
     return in;
 }
 
+const std::map<std::string, RGBAColor> stringToColorMap {
```

Comment:
Couldn't constexpr be used on a map ?

Reply:
Ok, it is not possible currently, but serge-sans-paille did it ! https://github.com/serge-sans-paille/frozen